fnirs_tapping/sub-01/nirs/sub-01_task-tapping_channels.tsv

```diff
@@ -0,0 +1,57 @@
+﻿name	type	source	detector	wavelength_nominal	units	low_cutoff	high_cutoff	description	sampling_frequency	status	status_description
+S1_D1 760	NIRSCWAMPLITUDE	S1	D1	760.0	V	0.0	3.90625	Near Infrared Spectroscopy (continuous wave)	7.8125	good	n/a
```

Comment:
How common is it for there to be different values in the type column? or are they always the same. Sorry, I probably missed some talk some where about this in the spec pr,

Reply:
Very common. Particularly for fNIRS data that is saved after conversion to haemoglobin. In this case, the type column is likely to include both `NIRCSWHBO` and `NIRSCWHBR`.